server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -374,7 +374,7 @@ private DivTag renderInfoCell(QuestionDefinition definition) {
         div()
             .withClasses("question-bank-info-cell")
             .with(
-                Icons.questionTypeSvg(definition.getQuestionType())
+                Icons.questionTypeSvgWithId(definition.getQuestionType())
```

Comment:
You want the link version here, not the Id.

Reply:
done! thanks for catching it!